cmd/todo-service/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	controller "github.com/daniyalumer/todo-list-go-chi/internal/controller"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+)
+
+func main() {
+	r := chi.NewRouter()
```

Comment:
Move the router configuration work in a separate function and return a router object and use in the `main` function